src/wallet/wallet2.cpp

```diff
@@ -2553,6 +2556,11 @@ bool wallet2::generate_chacha_key_from_secret_keys(crypto::chacha_key &key) cons
   crypto::generate_chacha_key(data, sizeof(data), key);
 
   memset(data, 0, sizeof(data));
```

Comment:
strange... why would we need to zero a local buffer right before leaving the scope???